src/autowiring/CoreThreadMac.cpp

```diff
@@ -11,8 +11,17 @@
 #include <sys/types.h>
 #include <unistd.h>
 #include <iostream>
+#include <pthread.h>
 #include THREAD_HEADER
 
+// Missing definitions from pthreads.h
+#if !defined(PTHREAD_MIN_PRIORITY)
+#define PTHREAD_MIN_PRIORITY  0
+#endif
+#if !defined(PTHREAD_MAX_PRIORITY)
+#define PTHREAD_MAX_PRIORITY 31
```

Comment:
It looks like 63 is max priority in user space according to https://developer.apple.com/library/mac/documentation/Darwin/Conceptual/KernelProgramming/scheduler/scheduler.html.
Why did you put 31?